### PR TITLE
Add the activeParticipants event to the notification service RN+android

### PR DESCRIPTION
**Pull request by kbetl-dlb**

### CODE REVIEW

File: android/src/main/java/io/dolby/sdk/comms/reactnative/eventemitters/RNNotificationEventEmitter.kt [hunk 1]
```diff
@@ -127,6 +149,7 @@ class RNNotificationEventEmitter(
     object ConferenceStatus : RNEvent("EVENT_NOTIFICATION_CONFERENCE_STATUS")
     object ParticipantJoined : RNEvent("EVENT_NOTIFICATION_PARTICIPANT_JOINED")
     object ParticipantLeft : RNEvent("EVENT_NOTIFICATION_PARTICIPANT_LEFT")
+    object ActiveParticipants : RNEvent("EVENT_NOTIFICATION_ACTIVE_PARTICIPANT_LEFT")
```
Comment (mateuszglapiak-dolby) [inline android/src/main/java/io/dolby/sdk/comms/reactnative/eventemitters/RNNotificationEventEmitter.kt:152]: I think it should be `EVENT_NOTIFICATION_ACTIVE_PARTICIPANTS `

**Review by mateuszglapiak-dolby — APPROVED** [review]

File: android/src/main/java/io/dolby/sdk/comms/reactnative/eventemitters/RNNotificationEventEmitter.kt (hunk 1 above)
Comment (kbetl-dlb) [reply android/src/main/java/io/dolby/sdk/comms/reactnative/eventemitters/RNNotificationEventEmitter.kt]: fixed

**Review by mateuszglapiak-dolby — DISMISSED** [review]

**Review by mateuszglapiak-dolby — APPROVED** [review]

**Comment by asadumair** [comment]
in ios?

**Comment by asadumair** [comment]
also for the listeners?

**Pull request merged**